client/src/common/components/Members/components/InviteNewUser.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import PendingButton from 'common/components/PendingButton';
+import { PreloaderTheme } from 'common/components/Preloader';
+
+const InviteNewUser = ({ email, onCancel, onInvite }) => (
+  <div className="invite-user">
+    <p className="invite-user__message">
+      User
+      <span className="invite-user__email">{` ${email} `}</span>
+      has no account in EOC app. Would you like to invite via email?
```

Comment:
`to send invite`
`EOC` should be some constant so that in future we won't have to change the used name in many places. Use some constant like APP_NAME